oak-segment-tar/src/main/java/org/apache/jackrabbit/oak/segment/file/tar/TarFiles.java

```diff
@@ -411,18 +413,28 @@ public void init() throws IOException {
         // iterates the indices in ascending order, but prepends - instead of
         // appending - the corresponding TAR readers to the linked list. This
         // results in a properly ordered linked list.
+        Stream.of(indices)
+                .parallel()
+                .map(index -> {
+                    try {
+                        if (readOnly) {
+                            return TarReader.openRO(map.get(index), tarRecovery, archiveManager);
+                        } else {
+                            return TarReader.open(map.get(index), tarRecovery, archiveManager);
+                        }
+                    } catch (IOException e) {
+                        log.warn("Unable to open TAR file: {}", map.get(index), e);
+                        throw new UncheckedIOException(e);
```

Comment:
how is the exception handling for this UncheckedIOException? And would it possible to pass some more information (for example the name of the file (it's logged before).